plutus-core/plutus-core/test/Evaluation/Spec.hs

```diff
@@ -45,15 +46,16 @@ type Term uni fun = PLC.Term TyName Name uni fun ()
 -}
 test_builtinsDon'tThrow :: TestTree
 test_builtinsDon'tThrow =
-    testGroup
-        "Builtins don't throw"
-        $ (\fun -> testGroup "for every version" $
-           (\ver ->
-                testPropertyNamed (render $ pretty fun <> pretty ver) (fromString . render $ pretty fun <> pretty ver) $
-                   prop_builtinEvaluation def fun gen f
-           -- perhaps using maxBound (with Enum, Bounded) is indeed better than Default for BuiltinVersions
-           ) <$> enumerate @(BuiltinVersion DefaultFun)
-          ) <$> enumerate @DefaultFun
+    testGroup "Builtins don't throw" $
+        enumerate @(BuiltinVersion DefaultFun) <&> \ver ->
+            testGroup (fromString . render $ "Version: " <> pretty ver) $
+                enumerate @DefaultFun <&> \fun ->
```

Comment:
The [style guide](https://github.com/input-output-hk/plutus/blob/master/STYLEGUIDE.adoc#avoid-multi-line-parenthesized-expressions) blesses you.

Reply:
Maybe it's worth adding `fmap (\x -> ...) xs` => `xs <&> \x -> ...` as an example then? Because otherwise it may seem like we insist on turning the former into `fmap f xs where f x = ...` (I got confused for a moment myself).

Reply:
> Horrible nitpicking, I know

not at all, it creates a better test output.

Reply:
I never use <&> , I always use <$> instead, I should start using that as well.

Reply:
>  `number_of_versions` < `number_of_functions`

Well it is right now, but wait till we get to version 900!